### PR TITLE
[CRBFUS-37] Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by IreneNakiyingi**
create a database for a transportation company

### CODE REVIEW

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Please effect the requested changes  @IreneNakiyingi

File: submissions/ireneNakiyingi/database/transportationdb.sql [hunk 1]
```diff
@@ -25,18 +23,17 @@ number_of_parents_children INT NOT NULL,
 number_of_siblings_spouses INT NOT NULL,
 point_of_embarkation VARCHAR(100) NOT NULL,
 PRIMARY KEY (id),
-FOREIGN KEY (id) REFERENCES registered_passengers(id)
+FOREIGN KEY (id) REFERENCES travel_id
```
Comment (ToluEnny24) [inline submissions/ireneNakiyingi/database/transportationdb.sql:26]: this is still wrong.
it should be FOREIGN KEY travel_id REFERENCES registered_passengers (id)

In english this means the column travel_id references the id column in the table registered_passengers.

File: submissions/ireneNakiyingi/database/transportationdb.sql [hunk 2]
```diff
@@ -25,18 +23,17 @@ number_of_parents_children INT NOT NULL,
...
+
 
 -- table in the event of an accident, status = alive/dead
 CREATE TABLE flight_status(
 id INT NOT NULL AUTO_INCREMENT,
 traveller_id INT NOT NULL,
 status VARCHAR(10) NOT NULL,
 PRIMARY KEY (id),
-FOREIGN KEY (id) REFERENCES registered_passengers(id)
+FOREIGN KEY (id) REFERENCES traveller_id
```
Comment (ToluEnny24) [inline submissions/ireneNakiyingi/database/transportationdb.sql:36]: Use the comment above for this too.
FOREIGN KEY traveller_id REFERENCES blahblah blah

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Review again. @IreneNakiyingi

File: submissions/ireneNakiyingi/database/transportationdb.sql (hunk 2 above)
Comment (IreneNakiyingi) [reply submissions/ireneNakiyingi/database/transportationdb.sql]: I've made the changes. sorry, totally misunderstood the previous correction.

**Pull request merged**